### PR TITLE
Fix pingpong teal test

### PR DESCRIPTION
**Pull request by egieseke**
## Summary

Updated the netgoal utility to only create one wallet per non-participating node to better concentrate Algos for use by pingpong. 

## Test Plan

Ran netgoal to update the scenario2 recipe and tested the result with the CICD performance pipeline.

### CODE REVIEW

**Review by tsachiherman — DISMISSED** [review]
looks good to me. Please make sure to regenerate all the scenarios with this change and include them in the PR.

**Comment by tsachiherman** [comment]
Changing the total supply can trigger some failures that would not be possible on mainnet. Are we truely ran out of money?

**Comment by onetechnical** [comment]
> Changing the total supply can trigger some failures that would not be possible on mainnet. Are we truely ran out of money?

+1. I'm still not a fan of increasing the amount of supply, as it indicates an unrealistic test (we would never increase mainnet supply).

File: test/testdata/deployednettemplates/recipes/scenario2/genesis.json [hunk 1]
```diff
@@ -8,1077 +8,1077 @@
...
 		{
 			"Name": "Wallet206",
-			"Stake": 0.46511627906976744,
+			"Stake": 3.3333333333333335,
 			"Online": false
 		},
 		{
 			"Name": "Wallet207",
-			"Stake": 0.46511627906976744,
+			"Stake": 3.3333333333333335,
```
[the remark this answers is not in the document]
Comment (egieseke) [reply test/testdata/deployednettemplates/recipes/scenario2/genesis.json:1041]: The rebalanced stake is computed in the function 
`func generateWalletGenesisData(wallets, npnHosts int)`
in `cmd/netgoal/generate.go`

**Review by algobarb — DISMISSED** [review]
Messaged Eric to learn more about the details. The results look correct on Datadog now so I think the last fix to rebalance stake fixed it! Changes look good to me.

**Comment by egieseke** [comment]
> looks good to me. Please make sure to regenerate all the scenarios with this change and include them in the PR.

Updated scenario1 and scenario3.

**Comment by egieseke** [comment]
> Changing the total supply can trigger some failures that would not be possible on mainnet. Are we truely ran out of money?

By rebalancing the stake between participating and non-participating nodes, increasing the total stake is not required.  Reverted total stake back to 10 billion Algos.

**Review by algobarb — APPROVED** [review]
Looks like Scenario1 results look ok with the rebalanced stake and removed wallets as well. Looks good to me.

**Comment by egieseke** [comment]
> > Changing the total supply can trigger some failures that would not be possible on mainnet. Are we truely ran out of money?
> 
> +1. I'm still not a fan of increasing the amount of supply, as it indicates an unrealistic test (we would never increase mainnet supply).

@onetechnical Good point.  Updated so that the total stake remains at 10 billion.  Split stake evenly between participating and nonparticipating nodes, which provides a sufficient amount for the teal pingpong tests.

File: cmd/netgoal/generate.go [hunk 2]
```diff
@@ -465,19 +466,29 @@ func saveGoalTemplateToDisk(template netdeploy.NetworkTemplate, filename string)
...
+	ratZero := big.NewRat(int64(0), int64(1))
+	ratHundred := big.NewRat(int64(100), int64(1))
 	data := gen.DefaultGenesis
+	totalWallets := wallets + npnHosts
+	data.Wallets = make([]gen.WalletData, totalWallets)
+	participatingNodeStake := big.NewRat(int64(100), int64(wallets))
+	nonParticipatingNodeStake := ratZero
 	if npnHosts > 0 {
-		wallets = 2 * wallets
+		// split participating an non participating stake evenly
```
Comment (bricerisingalgorand) [inline cmd/netgoal/generate.go:477]: Have we always been assuming an even split between participating and non participating nodes? I wonder if that should be configurable. It looks like that would be a new feature though so probably out of scope

**Review by bricerisingalgorand — APPROVED** [review]

**Pull request merged**